tests/run_tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+docker compose -f tests/docker-compose.yml up -d
+echo "---*** Running Odoo tests at_install ***---"
+docker compose -f tests/docker-compose.yml exec odoo ./odoo-bin -d test --addons-path=addons,odoo/addons \
+-i base,\
+account \
+--test-enable --test-tags=-post_install \
+--stop-after-init
+echo "---*** Running Odoo tests post_install ***---"
+docker compose -f tests/docker-compose.yml exec odoo ./odoo-bin -d test --addons-path=addons,odoo/addons \
+--test-enable --test-tags=/base\
+/account \
+--stop-after-init
+docker compose -f tests/docker-compose.yml down -v
```

Comment:
A little bit more robust and slightly more readable:

```suggestion
function cleanup() {
    docker compose -f tests/docker-compose.yml down -v
}

trap cleanup EXIT

docker compose -f tests/docker-compose.yml up -d

echo "---*** Running Odoo tests at_install ***---"
docker compose -f tests/docker-compose.yml exec odoo ./odoo-bin -d test --addons-path=addons,odoo/addons \
    -i base,account \
    --test-enable --test-tags=-post_install \
    --stop-after-init

echo "---*** Running Odoo tests post_install ***---"
docker compose -f tests/docker-compose.yml exec odoo ./odoo-bin -d test --addons-path=addons,odoo/addons \
    --test-enable --test-tags=/base,/account \
    --stop-after-init

docker compose -f tests/docker-compose.yml down -v
```

Reply:
:thinking: This isn't the latest version of this file. I must have accidentally reverted that...

Nonetheless, great suggestions. I will incorporate them.